go.mod

```diff
@@ -1,17 +1,8 @@
-module github.com/google/cel-spec
+module cel.dev/expr
```

Comment:
From IRL convo: not sure if this would make generated pb go files be recognized as `cel.dev/expr/expr`. I'll talk to to some go experts on my team if they know how it'll work. Otherwise, we can always just merge it and test in https://github.com/cncf/xds/pull/89.

Reply:
I think the main right now is `go-import` tag behavior. I'd probably go with merging this PR as is, and then experimenting a bit.

Reply:
@TristonianJones fyi. It's possible we may have to move the golang protos to root.

Reply:
So I did some experimentation yesterday - and it looks like unfortunately the protos have to leave at root. I made a fake domain, and served a meta go-import tag to https://github.com/sergiitk/cel-go-experiment. Then only way I made it work (download and build) was by placing the repo at the root.
Will attach my scripts in a moment.

Reply:
Ok, the scripts are in the repo, readme updated as well

Reply:
Thanks, moved golang gened files to root

Reply:
I just double checked and it looks like this will be okay since most repos only depend on cel-go directly and not both cel-go and cel-spec.